docs/git-commands.md

```diff
@@ -0,0 +1,147 @@
+# Git Command Guide
+The following is a number of scenarios that you will encounter while contributing to this repo. A convention in this document is to use `<< >>` to indicate text that you will need to replace when running the commands on your own machine.
+
+## Clone this REPO
+Before you begin, you will need to clone this repo
+```
+git clone https://github.com/MountSionCBSSecondary/SchoolWebsite.git
+```
+The above command will download a copy of the repo to your local machine. This copy of the repo will have the remote name of `origin`.
+
+
+## Add your own Remote
+Once you have the origin remote you will need to add a remote to your own fork of the repo. This can be done with the following command, just replace the placeholder with your own github name.
```

Comment:
Suggestiong => `Once you have the origin remote configured locally (i.e. your local copy knows where original repo is), you will need to configure your fork as a remote locally(i.e. your local copy knows where your fork is)`